modules/distr-tracing-config-security-ossm-cli.adoc

```diff
@@ -0,0 +1,93 @@
+////
+This module included in the following assemblies:
+service_mesh/v2x/ossm-reference-jaeger.adoc
+////
+:_content-type: PROCEDURE
+[id="distr-tracing-config-security-ossm-cli_{context}"]
+= Configuring distributed tracing security for service mesh from the command line
+
+You can modify the Jaeger resource to configure {JaegerShortName} security for use with {SMproductShortName} from the command line using the `oc` utility.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.  If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
+* The {SMProductName} Operator must be installed.
+* The `ServiceMeshControlPlane` deployed to the cluster.
+* You have access to the OpenShift CLI (oc) that matches your OpenShift Container Platform version.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
++
+[source,terminal]
+----
+$ oc login https://<HOSTNAME>:6443
+----
++
+. Change to the project where you installed the control plane, for example istio-system.
++
+[source,terminal]
+----
+$ oc project istio-system
+----
++
+. Run the following command to edit the Jaeger custom resource file, where `jaeger.yaml` is the name of your Jaeger custom resource.
++
+[source,terminal]
+----
+$ oc edit -n tracing-system -f jaeger.yaml
+----
++
+. Edit the `Jaeger` custom resource file to add the `htpasswd` configuration as shown in the following example.
+
+* `spec.ingress.openshift.htpasswdFile`
+* `spec.volmes`
+* `spec.volumeMounts`
++
+.Example Jaeger resource showing `htpasswd` configuratin
+[source,yaml]
+----
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+spec:
+  ingress:
+    enabled: true
+    openshift:
+      htpasswdFile: /etc/proxy/htpasswd/auth
+      sar: '{"namespace": "istio-system", "resource": "pods", "verb": "get"}'
+    options: {}
+    resources: {}
+    security: oauth-proxy
+  volumes:
+    - name: secret-htpasswd
+      secret:
+        secretName: htpasswd
+    - configMap:
+        defaultMode: 420
+        items:
+          - key: ca-bundle.crt
+            path: tls-ca-bundle.pem
+        name: trusted-ca-bundle
+        optional: true
+      name: trusted-ca-bundle
+  volumeMounts:
+    - mountPath: /etc/proxy/htpasswd
+      name: secret-htpasswd
+    - mountPath: /etc/pki/ca-trust/extracted/pem/
+      name: trusted-ca-bundle
+      readOnly: true
+----
++
+. Run the following command to apply your changes.
++
+[source,terminal]
+----
+$ oc apply -n tracing-system -f jaeger.yaml
```

Comment:
Since there could be variation in the filename, I think replacing `jaeger.yaml` with a user replaced variable as I mentioned above would be clearer. You probably don't need a callout in the second example.